src/care/CMakeLists.txt

```diff
@@ -57,11 +56,15 @@ set(care_headers
 set(care_sources
     care.cpp
     CHAICallback.cpp
-    LoopFuser.cpp
```

Comment:
I would keep this as well, have it compile to empty (and allow it to be extended in the event LoopFuser disabled still needs CPU implementation for a stripped down LoopFuser class.

Reply:
@robinson96 @liu15  Oh no, I meant to close this pull request a long time ago. I think a long time ago I tried to build with spack with the +loop_fuser on, and there was an incompatibility in loop fuser due to conflicting versions. We ended up just updating the versions instead of removing loop fuser from the build system if I recall, this was not needed

Reply:
@adayton1 Don't merge! I meant to close this pull request a long time ago. I think a long time ago I tried to build with spack with the +loop_fuser on, and there was an incompatibility in loop fuser due to conflicting versions. We ended up just updating the versions instead of removing loop fuser from the build system if I recall, this was not needed